db/migrations/20230207110912_create_orders_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE orders (
+  id int PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  user_id int NOT NULL,
+  price_amount int,
```

Comment:
여기도 총액을 표기하는 부분이기 때문에 decimal 타입 지정으로 수정해주세요!
```suggestion
  price_amount decimal(10, 2),
```